packages/twenty-front/src/modules/object-record/utils/getAvailableAggregationsFromObjectFields.ts

```diff
@@ -0,0 +1,51 @@
+import { FieldMetadataItem } from '@/object-metadata/types/FieldMetadataItem';
+import { AGGREGATE_OPERATIONS } from '@/object-record/record-table/constants/AggregateOperations';
+import { FieldMetadataType } from '~/generated-metadata/graphql';
+import { capitalize } from '~/utils/string/capitalize';
+
+type NameForAggregation = {
+  [T in AGGREGATE_OPERATIONS]?: string;
+};
+
+type Aggregations = {
+  [key: string]: NameForAggregation;
+};
+
+export const getAvailableAggregationsFromObjectFields = (
```

Comment:
This util should be split, we need to have a very simple util that just return the available operations for one fieldType and then we can build upon it more complex utils for an array / dictionary of fields.